src/tests/data/tls-policy/datagram.txt

```diff
@@ -1,23 +1,28 @@
-allow_tls10 = false
-allow_tls11 = false
 allow_tls12 = false
 allow_tls13 = false
-allow_dtls10 = false
 allow_dtls12 = true
+allow_ssl_key_log_file = false
 ciphers = ChaCha20Poly1305 AES-256/GCM AES-128/GCM
 macs = AEAD
 signature_hashes = SHA-512 SHA-384 SHA-256
 signature_methods = ECDSA RSA
 key_exchange_methods = ECDH DH
-key_exchange_groups = x25519 x448 secp256r1 brainpool256r1 secp384r1 brainpool384r1 secp521r1 brainpool512r1 ffdhe/ietf/2048 ffdhe/ietf/3072 ffdhe/ietf/4096 ffdhe/ietf/6144 ffdhe/ietf/8192
+key_exchange_groups = x25519 secp256r1 x25519/Kyber-512-r3/cloudflare x448 secp384r1 secp521r1 brainpool256r1 brainpool384r1 brainpool512r1 ffdhe/ietf/2048 ffdhe/ietf/3072
 allow_insecure_renegotiation = false
 include_time_in_hello_random = true
 allow_server_initiated_renegotiation = false
 hide_unknown_users = false
 server_uses_own_ciphersuite_preferences = true
 negotiate_encrypt_then_mac = true
+support_cert_status_message = true
+tls_13_middlebox_compatibility_mode = true
+accepted_client_certificate_types = X509
+accepted_server_certificate_types = X509
+hash_hello_random = true
+maximum_session_tickets_per_client_hello = 1
 session_ticket_lifetime = 86400
-dh_group = modp/ietf/2048
+reuse_session_tickets = false
+new_session_tickets_upon_handshake_success = 1
```

Comment:
Informational: Most of those are TLS 1.3 specific policies (except `hash_hello_random`). None of these will currently have an effect on DTLS 1.2. No objection to add them, though.